Dev/config/fontStyles.js

```diff
@@ -24,12 +24,6 @@ export default StyleSheet.create({
         color: colors.darkGrey
     },
 
-    smallTextStyleDarkGrey: {
```

Comment:
Just double checking, you removed all references to this font?

Reply:
It was defined twice in the file